### PR TITLE
Pass target-roots to `CHECK-PRIORITY`

### PR DESCRIPTION
**Pull request by karalekas**
Also, pass the `debug?` flag when sowing nodes!

### CODE REVIEW

File: src/operations/multireweight.lisp [hunk 1]
```diff
@@ -51,65 +51,70 @@
...
   ;;       access to the underlying node's Lisp object (or its type).
   (push (make-data-frame-multireweight :internal-pong nil)
         (process-data-stack supervisor))
   (with-slots (root-bucket source-root) pong
     (setf root-bucket (remove-duplicates root-bucket :test #'address=))
     (process-continuation supervisor
-                          `(CONVERGECAST-COLLECT-ROOTS ,(list source-root) ,root-bucket)
-                          `(CHECK-PRIORITY ,source-root)
-                          `(START-INNER-MULTIREWEIGHT)
+                          `(CONVERGECAST-COLLECT-ROOTS ,source-root ,root-bucket)
```
Comment (ecpeterson) [inline src/operations/multireweight.lisp:62]: My preference is that this still installs the whole "script" for the operation. The body of `CHECK-PRIORITY` can be wrapped by `(when aborting? ...)`, so then both commands get skipped if either of the two aborting conditions in this command are met.

**Review by ecpeterson — APPROVED** [review]
One complaint about uniformity, o/w looks good

File: src/operations/multireweight.lisp (hunk 1 above)
Comment (karalekas) [reply src/operations/multireweight.lisp:62]: Fair, but the output of `CONVERGECAST-COLLECT-ROOTS` is used as input for `CHECK-PRIORITY` -- previously we used a data frame to deal with that but my goal here is to explicitly pass this information as an argument to `CHECK-PRIORITY'. In that case, I don't see how to install the full script at once.

Comment (ecpeterson) [reply src/operations/multireweight.lisp:62]: I see. I agree that that's not possible, or not easy.

There's still a data frame there, though, and CHECK-PRIORITY appears to be local to this file. What are we avoiding? Something affected by changes downstream of these?

Comment (karalekas) [reply src/operations/multireweight.lisp:62]: We're enabling something downstream, yeah

Comment (ecpeterson) [reply src/operations/multireweight.lisp:62]: Ah, then what can be done 🤷‍♀️

**Pull request merged**